fuel/converters/celeba.py

```diff
@@ -0,0 +1,212 @@
+import os
+import zipfile
+
+import h5py
+import numpy
+from six.moves import range
+from PIL import Image
+
+from fuel.converters.base import check_exists, progress_bar
+from fuel.datasets import H5PYDataset
+
+IMAGE_FILE = 'img_align_celeba.zip'
+ATTRIBUTES_FILE = 'list_attr_celeba.txt'
+DATASET_FILES = [IMAGE_FILE, ATTRIBUTES_FILE]
+NUM_EXAMPLES = 202599
+TRAIN_STOP = 162770
+VALID_STOP = 182637
+
+
+def _initialize_conversion(directory, output_path, image_shape):
+    h5file = h5py.File(output_path, mode='w')
+    split_dict = {
+        'train': {
+            'features': (0, TRAIN_STOP),
+            'attributes': (0, TRAIN_STOP)},
```

Comment:
Why not just stick with `targets` for consistency with most other datasets? Otherwise it should probably be mentioned in the docstring somewhere.